src/univariate/continuous/tdist.jl

```diff
@@ -73,7 +73,11 @@ end
 
 @_delegate_statsfuns TDist tdist ν
 
-rand(d::TDist) = StatsFuns.RFunctions.tdistrand(d.ν)
+function rand(rng::AbstractRNG, d::TDist)
+    z = randn(rng)
+    ν = d.ν
```

Comment:
`ν = d.ν` is this step required? we could use `d.ν` directly in the line below

Reply:
I was just copying the style of all of the other functions in that file that all stored the variable locally before using it multiple times... I presumed that there was some indirection cost in using the fields? Happy to remove if not...

Reply:
I've done this now.